Makefile

```diff
@@ -291,6 +291,10 @@ test: fmt envtest ## Run Unit tests.
 .PHONY: test-integration
 test-integration: ## Run Integration tests.
 	go clean -testcache
+	cd config/bpfman-deployment && \
+	  sed -e 's@bpfman\.image=.*@bpfman.image=$(BPFMAN_IMG)@' \
```

Comment:
u could use `yq` to replace in place the image as well and avoid the tmp kustomize ?

Reply:
Yes, we discussed `yq` in the past, but decided we didn't want to add another tool. We are already doing something similar in the examples Makefile.

Reply:
just FYI `sed` require some magic to work on mac os, probably something like
```bash
ifeq (,$(shell which gsed 2>/dev/null))
SED ?= sed
else
SED ?= gsed
endif
```
then use `SED` everywhere

Reply:
Good to know. I don't think anyone has tried to build/run on a MAC, but should probably be added to the list.